app/pandas.ipynb

```diff
@@ -345,22 +454,40 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# use + operator here"
+    "s1 = pd.Series([1, 2, 3, 4, 5])\n",
+    "s2 = pd.Series([6, 7, 8, 9, 10])\n",
+    "addition = s1 + s2\n",
+    "print(addition)"
    ],
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-11-11T19:08:06.504634Z",
+     "start_time": "2024-11-11T19:08:06.498902Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0     7\n",
+      "1     9\n",
+      "2    11\n",
+      "3    13\n",
+      "4    15\n",
+      "dtype: int64\n"
+     ]
+    }
+   ],
+   "execution_count": 21
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "outputs": [],
-   "source": [
-    "# use - operator here"
-   ],
+   "source": "subtraction = s1 - s2",
```

Comment:
The subtraction operation is defined but not executed. You should print the result to verify the operation.